### PR TITLE
string article

### PR DESCRIPTION
**Pull request by darlanmendonca**

### CODE REVIEW

**Comment by darlanmendonca** [comment]
@khaosdoctor @OdineiRibeiro give a review pls

File: content/en/string.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+var phrase = 'You know nothing';
+var username = 'John';
+```
+
+So better!! Now we see how to work with these strings.
+
+First, the username are wrong, the write is John Snow. Then, to overwrite the value of variable, 
+
+Primeiro, o username está errado, o correto seria John Snow. Então, para sobrescrever o valor da variável, just what we call the variable name again.
+
```
Comment (odineiramone) [inline content/en/string.md:26]: Why you misturou english and portuguesse nas frases?

File: content/en/string.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+## String
+
+One type of variables that have Javascript is the type String. String is the same as text. Technically, is a set of characters :D. To create strings, we use quotes or double quotes. See example:
+
+```js
+var phrase = 'You know nothing';
+var username = "John"
```
Comment (odineiramone) [inline content/en/string.md:11]: Have any relevant difference between them `'` and `"`?

Comment (darlanmendonca) [reply content/en/string.md:11]: technically not. but the patterns determine single quotes to be used

File: content/en/string.md (hunk 1 above)
Comment (darlanmendonca) [reply content/en/string.md:26]: hahua, good, my bad

**Comment by odineiramone** [comment]
:+1:

File: content/en/string.md (hunk 2 above)
Comment (khaosdoctor) [reply content/en/string.md:11]: Hmm, I think we should standardize this...

**Review by khaosdoctor — APPROVED** [review]
Made some changes, I think this is good to go

**Pull request merged**